README.md

```diff
@@ -47,11 +47,18 @@ NOTE:
 
 Status: **GA**
 
-## Supported CO platforms
+## Known to work CO platforms
 
 Ceph CSI drivers are currently developed and tested **exclusively** on Kubernetes
-environments. There is work in progress to make this CO independent and thus
-support other orchestration environments in the future.
+environments.
+
+| Ceph CSI Version | Container Orchestrator Name | Version Tested|
+| -----------------| --------------------------- | --------------|
+| v3.4.0 | Kubernetes | v1.20, v1.21, v.22|
+| v3.3.0 | Kubernetes | v1.20, v1.21, v.22|
```

Comment:
should we not list OpenShift too?

Reply:
@nixpanic actually I thought about it, but considering we dont have a CI for that, I left it out. if we are in agreement to list, we can list that too..

Reply:
I think its better to not list in this upstream repo, as we dont have a CI to verify against . @nixpanic @Madhu-1 wdyt?

Reply:
I think it is fine to mention OpenShift, users should be able to deploy ceph-csi on it. If there are changes needed to the Helm Charts or `deploy/` examples, we should probably document/add that. Not sure what needs changing though, I usually deploy with Rook, and that seems to work fine?

Reply:
Rook has the required SCC changes which are required to run in openshift but we have not documented the changes in cephcsi. IMO we can add OCP to the list once we add the required documentation

Reply:
I agree with @Madhu-1 here, compared to Rook repo, we only list kube templates here and cover it in the documentation . Once we have examples and documentation we could add openshift.